jobs/initial_data/__init__.py

```diff
@@ -1,14 +1,19 @@
 """Initial data required for core sites."""
+
 from nautobot.apps.jobs import register_jobs
 
 from nautobot_design_builder.design_job import DesignJob
+from nautobot.extras.jobs import IntegerVar, StringVar
 
 from .context import InitialDesignContext
 
 
 class InitialDesign(DesignJob):
     """Initialize the database with default values needed by the core site designs."""
+
     has_sensitive_variables = False
+    routers_per_site = IntegerVar(min_value=1, max_value=6, default=2)
+    custom_description = StringVar(default="custom description for us-west-1")
```

Comment:
this was used to explain the protection features, but I guess we could skip it as there are other ways for that

Reply:
yeah, I'm not sure we should have `custom_description` here. It is only used in one place in the design.